Core/include/Acts/Propagator/EigenStepper.ipp

```diff
@@ -217,12 +217,11 @@ Acts::Result<double> Acts::EigenStepper<E, A>::step(
     }
 
     // Compute and check the local integration error estimate
-    error_estimate =
+    errorEstimate =
```

Comment:
I think it is fine to assume, that `errorEstimate` is non-negative, since
- `h2` = h^2 >= 0
- ().norm >= 0
- abs() >= 0
so we don't need to ckeck for the `sqrt()` below.

But how can we ensure that it is `!= 0` or at least large enough to not mess up the divison?

Reply:
that is actually ensured by the `std::max(1e-20, errorEstimate)` I accidentally removed